### DISCUSSION TITLE
Inquiry Regarding Changing Bottom Icon in Flet Application

### DISCUSSION THREAD

**Discussion by harisabdul** [flet-dev/flet#2880] [General]
Dear Flet Developer,

I hope this email finds you well. My name is [Your Name], and I am a developer currently working on a project utilizing the Flet library for Flutter applications. Firstly, I would like to express my gratitude for the excellent toolset your team has provided; it has greatly facilitated my development process.

I am reaching out to inquire about a specific issue I encountered while customizing my application. I have successfully changed the favicon icon within my application, but unfortunately, the bottom icon displayed remains unchanged. I have thoroughly reviewed the documentation and explored various avenues within the library but have not yet found a solution to this issue.

Could you kindly provide guidance on how to change the bottom icon in a Flet application to a custom icon? Any pointers, code snippets, or documentation references would be greatly appreciated.

Thank you very much for your time and assistance. I understand that you may be busy, but any help you could offer in resolving this matter would be immensely beneficial to my project.

Looking forward to your response.

Warm regards,

HarisAbdul

**Comment by ndonkoHenri** [comment]
It depends on the command you use to package your application.
If you use `flet pack`: https://flet.dev/docs/guides/python/packaging-desktop-app#customizing-package-icon
If you use `flet build`: https://flet.dev/docs/guides/python/packaging-app-for-distribution#icons

**Comment by harisabdul** [comment]
Enhancement Request: Icon Display Issue in Executable File


Dear flet developers,

I trust this email finds you well.

I am writing to bring to your attention a minor yet impactful issue
regarding the display of icons in our executable files. Recently, I
utilized the flet pack command to include an icon file, which successfully
displayed the desired icon at the bottom of the application window.
However, I noticed that the executable file itself continues to display the
default PyInstaller icon, rather than the intended one.

While this discrepancy does not hinder functionality, I believe that having
a consistent visual representation throughout the application would greatly
enhance user experience and branding consistency.

Would it be possible to investigate this matter further and implement a
solution that ensures the designated icon is displayed prominently across
all aspects of the application, including the executable file?

Your attention to this matter is greatly appreciated. Should you require
any further information or assistance from my end, please do not hesitate
to reach out.

Thank you for your time and consideration.

Warm regards,
